source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRColumnRenderer.mm

```diff
@@ -69,15 +69,16 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
 
     std::shared_ptr<BaseActionElement> selectAction = columnElem->GetSelectAction();
     // instantiate and add tap gesture recognizer
-    UITapGestureRecognizer * tapGestureRecognizer =
+    UILongPressGestureRecognizer * gestureRecognizer =
         [ACRTapGestureRecognizerFactory getTapGestureRecognizer:viewGroup
                                              rootViewController:vc
+                                                     targetView:column
                                                   actionElement:selectAction
                                                          inputs:inputs
                                                      hostConfig:config];
-    if(tapGestureRecognizer)
+    if(gestureRecognizer)
     {
-        [column addGestureRecognizer:tapGestureRecognizer];
```

Comment:
>tapGestureRecognizer [](start = 37, length = 20)

You don't want these renderers to respond to taps anymore?

Reply:
That's correct. I need to change its name to LongPressGestureRecognizerFactory

---
In reply to: [164590401](https://github.com/Microsoft/AdaptiveCards/pull/1075#discussion_r164590401) [](ancestors = 164590401)